index.html

```diff
@@ -234,20 +234,20 @@ <h3>Latest news</h3>
           <!-- News box Start -->
           <div class="news-box">
             <div class="news-box-title">
-              <span>18/07/2023</span>
+              <span>27/09/2023</span>
             </div>
             <div class="news-box-content">
-              <p>We just released a new patch on top of Uyuni 2023.04 that provides new GPG keys and security fixes. Check the <a href="https://lists.opensuse.org/archives/list/announce@lists.uyuni-project.org/thread/OQYCQDHPHWU2QXCEHX3SHQI5HKPPM5N2/" target="_blank">announcement</a> to get all the details about the fix and instructions to install the patch. We strongly recommend you update as soon as possible.</p>
+              <p>Uyuni 2023.09 is now available! The OS for the Server and Proxies is now openSUSE Leap 15.5, so major upgrade procedures will be needed. Salt is now updated to 3006.0. Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```

Comment:
```suggestion
              <p>Uyuni 2023.09 is now available! The OS for the Server and Proxies is now openSUSE Leap 15.5, so major upgrade procedures will be needed. Salt is now updated to 3006.0 and it will require a special update procedure. Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```